layers/vk_layer_utils.cpp

```diff
@@ -28,6 +28,12 @@
 #include "vulkan/vulkan.h"
 #include "vk_layer_config.h"
 
+// Include new / delete overrides if using mimalloc. This needs to be include exactly once in a file that is
+// part of the layer utils library.
+#if defined(USE_MIMALLOC)
```

Comment:
Ensure MIMALLOC is only used on WIN64 builds.

```suggestion
#if defined(USE_MIMALLOC) && defined(_WIN64)
```